apisix/stream/router/ip_port.lua

```diff
@@ -176,13 +193,31 @@ function _M.routes()
     return user_routes.values, user_routes.conf_version
 end
 
+local function address_checker(item)
```

Comment:
This function doesn't only check address.
```suggestion
local function checker(item)
```